### DISCUSSION TITLE
Create Code Of Conduct

### DISCUSSION THREAD

**Discussion by logankilpatrick** [aesara-devs/aesara#861] [Ideas]
Hey folks! I wanted to suggest you all create a CoC. You are welcome to adopt the NumFOCUS https://numfocus.org/code-of-conduct so long as you add a clear way for folks to report CoC violations to the team.

**Comment by logankilpatrick** [comment]
CC @brandonwillard @ricardoV94 @michaelosthege @kc611 It would be great to move quickly on this

**Comment by kc611** [comment]
Started a draft over here (Adopted the NumFOCUS CoC): https://docs.google.com/document/d/1RCFyYyCRGRkTp8Mmt_WMxutYFOGWEGaACD2jmjeY6iM/edit?usp=sharing

cc @brandonwillard for edits and suggestions.

**Comment by brandonwillard** [comment]
A PR has been opened for this: https://github.com/aesara-devs/aesara/pull/932.
